pkg/limitador/k8s_objects.go

```diff
@@ -196,3 +198,17 @@ func ownerRefToLimitador(limitador *limitadorv1alpha1.Limitador) metav1.OwnerRef
 		UID:        limitador.UID,
 	}
 }
+
+func storageType(storage *limitadorv1alpha1.Storage) string {
+	if storage != nil {
+		return string(storage.Type)
+	}
+	return string(limitadorv1alpha1.StorageTypeInMemory)
```

Comment:
Am I missing something here, but this will return `in-memory` and [pass that as a subcommand](https://github.com/Kuadrant/limitador-operator/pull/48/files#diff-41181542228742e869e5d34e69e5c72a4bc8f1a5ddc93c6199470479fc3b73fbR97) to `limitador-server`, but the supported values are:

```
STORAGES:
    memory          Counters are held in Limitador (ephemeral)
    redis           Uses Redis to store counters
    redis_cached    Uses Redis to store counters, with an in-memory cache
```

Reply:
oops, right, so `in-memory`is wrong and should be just `memory` which is the value supported. Thanks!